.devcontainer/devcontainer.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "modulo",
+  "privileged": true,
+  "remoteUser": "ros2",
+  "image": "ghcr.io/aica-technology/modulo:development",
```

Just a thought, it might be more convenient to let vscode build the image for you, rather than needing to use the separate build script.

```suggestion
  "build": {
    "Dockerfile": "../Dockerfile",
    "context": "..",
    "target": "development"
  }
```

Optionally you could also include `"args": { "CL_VERSION": "v7.1.0" } `

---

Yeah, for now it's still useful to have the script for easily doing build, test and SSH server. But the development layer doesn't bring that much on its own.